release.json

```diff
@@ -103,4 +103,4 @@
     "dca-1.9.0": {
         "SECURITY_AGENT_POLICIES_VERSION": "v0.3"
     }
-}
+}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ we should figure out where this keeps getting changed and fix it 😆

Reply:
Indeed, I'll take a look

Reply:
Definitely not pressing, as it's been happening forever (as I remember)

Reply:
It's been bugging me for a while, and I regret not taking a look before since it was an easy change: https://github.com/DataDog/datadog-agent/pull/26561